app/src/main/java/fr/free/nrw/commons/contributions/MainActivity.java

```diff
@@ -64,6 +68,9 @@
     @Inject
     QuizChecker quizChecker;
 
+    @Inject
+    CommonsAppAdapter appAdapter;
```

Comment:
We don't need to access the AppAdapter anywhere else. It doesn't need to be injectable.